Corgibytes.Freshli.Cli/CommandRunners/Cache/PrepareCacheActivity.cs

```diff
@@ -0,0 +1,67 @@
+using System;
+using Corgibytes.Freshli.Cli.DataModel;
+using Corgibytes.Freshli.Cli.Extensions;
+using Corgibytes.Freshli.Cli.Functionality;
+using Corgibytes.Freshli.Cli.Functionality.Engine;
+using Corgibytes.Freshli.Cli.Resources;
+
+namespace Corgibytes.Freshli.Cli.CommandRunners.Cache;
+
+public class PrepareCacheActivity : IApplicationActivityEngine, IApplicationActivity
```

Comment:
This class shouldn't implement `IApplicationActivityEngine`, just `IApplicationActivity`. This should be done before we merge.